e2e-tests/tfm/cypress/e2e/journeys/utilisation-reports/generate-keying-sheet.spec.js

```diff
@@ -45,15 +46,15 @@ context('PDC_RECONCILE users can generate keying data', () => {
     .build();
 
   const ONE_YEAR_AGO = subYears(TODAY, 1);
-  const ONE_YEAR_FROM_CURRENT_REPORT_PERIOD_START = addYears(START_OF_CURRENT_REPORT_PERIOD, 1); // days left in cover period = 365
+  const ONE_YEAR_FROM_CURRENT_REPORT_PERIOD_END = addDays(END_OF_CURRENT_REPORT_PERIOD, 365); // days left in cover period = 365
```

Comment:
I didn't want to have to worry about leap years so changed this

Reply:
Ah nice catch!